src/edge/tasks.py

```diff
@@ -5,6 +5,12 @@
 from django.db import transaction
 
 
+@shared_task
+def build_genome_fragment_indices(genome_id):
+    genome = Genome.objects.get(pk=genome_id)
+    genome.indexed_genome()
```

Comment:
Cool. So this is different from the build blastdb for the genome, right? also where is the index stored?

Reply:
The indices are per-fragment, and maps sequence bps to a bp numeric number. It's stored in the database.